inst/tests/tests.Rraw

```diff
@@ -17334,9 +17337,10 @@ test(2183.74, melt(DTid, measure.vars=measure(letter, number, multiple.keyword=a
 test(2183.75, melt(DTid, measure.vars=measure(letter, number, multiple.keyword=NA_character_, pattern="([ab])([12])")), error="multiple.keyword must be a character string")
 test(2183.76, melt(DTid, measure.vars=measure(letter, number, multiple.keyword="", pattern="([ab])([12])")), error="multiple.keyword must be a character string with nchar>0")
 test(2183.77, melt(DTid, measure.vars=measure(letter, cols, pattern="([ab])([12])")), error="group names specified in ... conflict with measure argument names; please fix by changing group names: [cols]")
-test(2183.78, melt(DTid, measure.vars=measure(letter, cols=as.integer, pattern="([ab])([12])")), error="cols must be a character vector of column names")
+test(2183.78, melt(DTid, measure.vars=measure(letter, cols=as.integer, pattern="([ab])([12])")), error="user should not provide cols argument to measure")
 test(2183.79, melt(DTid, measure.vars=measure(letter, number, pattern=as.integer)), error="pattern must be character string")
 test(2183.80, melt(DTid, measure.vars=measure(letter, number, sep=as.integer)), error="sep must be character string")
+test(2183.81, data.table:::measure(cols=as.integer), error="cols must be a character vector of column names")
```

Comment:
Prefer a local mask for unexported functions, see the top of this file (sorry, can't permalink easily on this massive file, but it's right at the top)

Reply:
this will not get tested during `cc()` afaict

Reply:
I see the block of local definitions for un-exported functions, but I don't think it will work in this case, because there are a few tests, for example 2183.01, which require defining another object named measure (would over-write) so I think it is better to just explicitly use data.table:::measure, is that ok?

Reply:
Not fine. data.table::: can be used only at the too of the test script

Reply:
ok, I fixed it!